docs/admin/kubeadm.md

```diff
@@ -460,6 +495,49 @@ started.
 Once the cluster is up, you can grab the admin credentials from the master node
 at `/etc/kubernetes/admin.conf` and use that to talk to the cluster.
 
+Note that this style of bootstrap has some relaxed security guarantees because
+it does not allow the root CA hash to be validated with
+`--discovery-token-ca-cert-hash` (since it's not generated when the nodes are
+provisioned). For details, see the [security model](#security-model)
+
+## Security model {#security-model}
```

Comment:
Suggestion, but only if you think it's helpful (since formatting might take a while): Might be better for visibility if you broke each of the three sections into "Advantages" and "Disadvantages", e.g.:

> ### Token-based discovery with CA pinning
> *This is the default mode in Kubernetes 1.8.*
>
> **Advantages**
> * It allows bootstrapping nodes...
> * It's also convenient to execute manually...
>
> **Disadvantages**
> * The CA hash is not normally known...

Reply:
Thanks! I took a shot at this in the latest commit (just pushed).